### DISCUSSION TITLE
[Suggestion] Profile and Account Upgrades

### DISCUSSION THREAD

**Discussion by metalcupcake5** [HypixelDev/PublicAPI#309] [Ideas]
I suggest adding the new upgrades added in SkyBlock Patch 0.9, shown below
![image](https://user-images.githubusercontent.com/43897385/92669289-fa4c3980-f2c5-11ea-8b06-b02aa9da2d1b.png)

**Comment by shadowtime2000** [comment]
Also player fame and contributions to city projects.

**Comment by doej1367** [comment]
and a gems_spent_counter would be nice

**Comment by MartinNemi03** [comment]
I think we can see profile upgrades now, but yeah, account upgrades would be nice to view
